README.md

```diff
@@ -8,16 +51,222 @@ so don't try to use it for anything close to a production environment.
+NAME        HOST/PORT                                                   PATH                                                   SERVICES                    PORT   TERMINATION          WILDCARD
+api-2fv99   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /                                                      metadata-server             api    reencrypt/Redirect   None
+api-cbvbz   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/resourceTypes        resource-server             api    reencrypt/Redirect   None
+api-v92jb   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/deploymentManagers   deployment-manager-server   api    reencrypt/Redirect   None
+api-xllml   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/resourcePools        resource-server             api    reencrypt/Redirect   None
+```
+
+The operator by default creates a default inventory CR in the oran-o2ims namespace:
+
+```bash
+$ oc get inventory -n oran-o2ims
+NAME      AGE
+default   4m20s
+```
+
+> :warning: Currently, the following components are enabled by default. 
+
+```sh
+$ oc get inventory -n oran-o2ims -oyaml
+```
+```yaml
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  creationTimestamp: "2024-11-07T09:26:12Z"
+  generation: 1
+  name: default
+  namespace: oran-o2ims
+  resourceVersion: "279507769"
+  uid: aeb85580-f7ca-4861-8c38-3f57902b1617
+spec:
+  alarmSubscriptionServerConfig:
+    enabled: false
+  deploymentManagerServerConfig:
+    backendType: regular-hub
+    enabled: true
+  metadataServerConfig:
+    enabled: true
+  resourceServerConfig:
+    enabled: true
+```
+
+### <a name='Run'></a>Run
+
+The `Inventory` custom resource (CR) is the way to configure the different components of the IMS operator. The default Inventory CR that is created by the system is sufficient to enable the basic API functionality. 
+
+The end user must edit the default Inventory CR to enable communication with the SMO instance after the SMO has been configured to accept connections from this O-Cloud. Specifically, the `cloudID` and `smo` attributes will need to be configured appropriately. The `cloudID` is the globally unique UUID value assigned to this O-Cloud by the SMO.
+
+```yaml
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/created-by: oran-o2ims
+    app.kubernetes.io/instance: orano2ims-sample
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: orano2ims
+    app.kubernetes.io/part-of: oran-o2ims
+  name: default
+  namespace: oran-o2ims
+spec:  
+  smo:
+    url: http://smo.example.com
+    registrationEndpoint: /mock_smo/v1/ocloud_observer
+  #image: quay.io/NAMESPACE/CONTAINER_IMAGE // uncomment if you want to use your built image otherwise will be populated with the default image.
```

Comment:
The comment here is not accurate.  If using `make deploy` the image used will be the user's development image.  The only reason to use this is if for some reason someone wanted to use a completely different image.  I would prefer if we removed this comment.